MREUnityRuntime/MREUnityRuntimeLib/App/MixedRealityExtensionApp.cs

```diff
@@ -943,12 +943,25 @@ private void OnSetAnimationState(SetAnimationState payload)
 
 
 
+        private static Dictionary<Guid, AudioSource> _soundInstances = new Dictionary<Guid, AudioSource>();
+        private static List<Guid> _unpausedSoundInstances = new List<Guid>();
+        private int _soundStoppedCheckIndex = 0;
```

Comment:
Normally I would say that these need to go at the top of the class, but nobody wants to scroll past 900 lines to get to them. This class is getting too big.